includes/Traits/Amend_Check_Result.php

```diff
@@ -41,6 +42,7 @@ protected function add_result_message_for_file( Check_Result $result, $error, $m
 				'line'   => $line,
 				'column' => $column,
 				'link'   => $this->get_file_editor_url( $result, $file, $line ),
+				'check'  => $this->get_name(),
```

Comment:
Does it optional? What about admin screen?

Reply:
This method is supposed to be called by Check only. So I believe we can assume it won't be empty.